src/processor/tree/mod.rs

```diff
@@ -65,6 +72,13 @@ impl Processor for TreeProcessor {
             // Update snapshot values.
             snapshot.latest_l2_block_number = block.l2_block_number;
             snapshot.index_to_key_map = self.tree.index_to_key_map.clone();
+
+            let after = Instant::now();
+            let duration = after.duration_since(before);
+            if metric.add(duration) > 10 {
+                let avg = metric.renew();
+                tracing::info!("BACKGROUND: avg snapshot {}", avg);
```

Comment:
```suggestion
                tracing::debug!("BACKGROUND: avg snapshot {}", avg);
```
`tracing::info` is a bit too noisy with this in my opinion.

Reply:
...and debug has other messages, so let's use a custom target for metrics...